_posts/2025-05-28-#32051.md

```diff
@@ -90,16 +90,188 @@ afterwards.
 
 6. One reviewer
    [suggested](https://github.com/bitcoin/bitcoin/pull/32051#pullrequestreview-2691505746)
-   clearing bock requests instead of disconnecting peers. How would you
+   clearing block requests instead of disconnecting peers. How would you
    implement this?
 
 7. How is the test coverage for the code affected by this change? Can
    you think of any tests that would be worthwhile adding?
 
+## Erratum
+
+During the meeting, I was wrong and stringintech was right; the addnode code may
+indeed have been introduced by Satoshi. See this commit: `git show e66ec79b`
 
-<!-- TODO: After a meeting, uncomment and add meeting log between the irc tags
 ## Meeting Log
 
 {% irc %}
+17:00:19 <jonatack> #startmeeting
```

Comment:
I think you need to strip the seconds, as it's rendering incorrectly:

<img width="978" alt="image" src="https://github.com/user-attachments/assets/68fdeebc-cce6-493c-aeda-72e39dd944ca" />